docker-hub/builds/index.md

```diff
@@ -277,6 +277,11 @@ You could also use capture groups to build and label images that come from vario
 
 `/(alice|bob)-v([0-9.]+)/` -->
 
+### Build image with Buildkit
+
+You can enable the Buildkit builder by setting the environment variable `DOCKER_BUILDKIT=1`([enviroment variables](index.md#environment-variables-for-builds)) in the Configure automated build section ([configure builds](index.md#configure-automated-build-settings)).
```

Comment:
- Given it's the same page, we don't strictly need the `index.md` (and would work around some bugs in Jekyll/plugins).
- I think we could just "Configure automated build section" as the caption for the link here
- s/Buildkit/BuildKit/ ? (capital `K`)
- Wrap to 80 chars 🤗 

```suggestion
You can enable the BuildKit builder by setting the `DOCKER_BUILDKIT=1`
[environment variable](#environment-variables-for-builds) in the
[Configure automated build settings](#configure-automated-build-settings) section.
```